docs/developer_tools/Embedded_App_SDK.md

```diff
@@ -528,15 +528,17 @@ Locks the application to specific orientations in each of the supported layout m
 
 #### Required Scopes
 
-- guilds.members.read
+No scopes required
 
 #### Usage
 
 ```js
+import {Common} from '@discord/embedded-app-sdk';
+
 await discordSdk.commands.setOrientationLockState({ 
-  lock_state: 'landscape', 
-  picture_in_picture_lock_state: 'landscape', 
-  grid_lock_state: 'unlocked'
+  lock_state: Common.OrientationLockStateTypeObject.LANDSCAPE,
+  picture_in_picture_lock_state: Common.OrientationLockStateTypeObject.LANDSCAPE,
+  grid_lock_state: Common.OrientationLockStateTypeObject.UNLOCKED
```

Comment:
A bit more verbose than I'd like but it's "state of the art" 😅

Reply:
123

Reply:
123